pkg/util/metamorphic/constants.go

```diff
@@ -8,7 +8,7 @@
 // by the Apache License, Version 2.0, included in the file
 // licenses/APL.txt.
 
-package util
+package metamorphic
```

Comment:
Consider adding a package-level doc comment explaining at a high level what the package does / what metamorphic constants are.